### ISSUE TITLE
Install Ruby on the build machine if it is not preinstalled

### ISSUE THREAD

**Issue by steven1046** [OpenLiberty/openliberty.io#1567]
Most of the time Ruby is already installed on the machines running the builds but every now and then we'll get a machine that fails immediately due to no Ruby: 
```
Installing ruby packages...
./scripts/build_gem_dependencies.sh: line 2: gem: command not found
./scripts/build_gem_dependencies.sh: line 3: gem: command not found
./scripts/build_gem_dependencies.sh: line 4: gem: command not found
./scripts/build_gem_dependencies.sh: line 5: gem: command not found

Finished: ERRORED
```
We should check if Ruby exists and install it so the builds don't fail.

**Comment by steven1046** [comment]
This only occurs on the IBM Cloud

**Comment by NottyCode** [comment]
I think this is fixed now we are running using the 2.8 delivery pipeline rather than the 1.0 one.

**Comment by steven1046** [comment]
Ok, we can close and reopen it if it reoccurs.
